zeppelin-server/pom.xml

```diff
@@ -43,19 +43,19 @@
       <dependency>
         <groupId>org.scala-lang</groupId>
         <artifactId>scala-library</artifactId>
-        <version>2.10.4</version>
+        <version>2.11.8</version>
```

Comment:
When I run zeppelin on my laptop with bin/zeppelin.sh, the scalding RemoteInterpreterServer process classpath has zeppelin-server classes before scalding. This causes 2.10.4 scala libraries to be used resulting in the errors when I run scalding which uses 2.11.8 scala libraries. Since scala is used only for scalatest in zeppelin-server and the tests run fine when I use scalatest_2.11, I thought I would keep this change and get your feedback. Let me know if you want me to revert this change. The scalding interpreter runs fine in production where we use zeppelin-distribution created by "mvn package -Pbuild-distr".